stan/math/prim/scal/prob/beta_lpdf.hpp

```diff
@@ -101,7 +97,7 @@ typename return_type<T_y, T_scale_succ, T_scale_fail>::type beta_lpdf(
 
   for (size_t n = 0; n < length(y); n++) {
     if (include_summand<propto, T_y, T_scale_succ>::value)
-      log_y[n] = log(value_of(y_vec[n]));
+      log_y[n] = std::log(value_of(y_vec[n]));
```

Comment:
Same comment again.  These can't be `std::` here because the values might not be `double`.

Reply:
Reverted.